src/Mod/Sketcher/Gui/DrawSketchController.h

```diff
@@ -564,6 +564,10 @@ class DrawSketchController
         onViewIndexWithFocus = 0;
 
         configureOnViewParameters();
+
+        if (init) {
+            handler->moveCursorToSketchPoint(prevCursorPosition);

```

Comment:
The bug described in #11307 (see https://youtu.be/24UYGO5PgjU) happens here. This moveCursorToSketchPoint is buggy

Reply:
Besides the bug in moveCursorToSketchPoint, is this necessary here? Why are you doing this?

Reply:
Replied in the other PR.